CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+## 1.10.4 - 2024-07-08
+
+### Added
+
+- `Swift` terminal profile
+- Set Swift environment variables in the integrated terminal
+- Show macro expansions
```

Comment:
nit: Show macro expansions via Quick Action

Reply:
Can we remove reference to the macro expansion. It is reliant on sourcekit-lsp changes that won't make it into 6.0.

Reply:
Removed

Reply:
I think it is better this is mentioned in any GSoC posts on the forums